### DISCUSSION TITLE
Volume Mount error: fsck found errors on device but could not correct them

### DISCUSSION THREAD

**Discussion by purushothamkdr453** [rook/rook#14923] [Q&A]
Hi team,

i am getting below error while describing statefulset pod.

**Error:**

Internal desc = 'fsck' found errors on device /dev/rbd27 but could not correct them: fsck from util-linux 2.32.1

Moredetails below.

**Cluster Setup**

6 node kubernetes cluster(rke cluster)

**statefulset pod**

pod is scheduled on 6th node of cluster.

kubectl describe pod \<podname\> -n \<namespace\>

<img width="945" alt="fsck-error" src="https://github.com/user-attachments/assets/08228a94-0496-4ca5-88b3-2ff09d93f954">

I went to 6th node(where pod is scheduled) and looked for /dev/rdb27 but couldnt find anything under(fdisk -l, df -h output).

**CSI image of PV**

<img width="950" alt="2" src="https://github.com/user-attachments/assets/621ce257-4815-4928-9e86-7766cacbf98f">

**No mapped devices for csi**

I have exec into toolbox pod and looked for a device corresponding to csi.

<img width="656" alt="4" src="https://github.com/user-attachments/assets/5f88a798-8444-4bf2-ab35-4fac84dd5238">

Any idea what this issue is?

if there is no device corresponding to csi image then from where it is getting the data and showing it as duplicate or bad blocks. 

Kindly do the needful.

**Comment by Madhu-1** [comment]
Try searching in the existing issue you will find a way to run fsck like this one https://github.com/ceph/ceph-csi/issues/1285. CSI will unmap the rbd image if the mount fails for any reason
